copycat/runtime/src/main/java/org/apache/kafka/copycat/runtime/WorkerSinkTask.java

```diff
@@ -108,6 +108,9 @@ public void close() {
     /** Poll for new messages with the given timeout. Should only be invoked by the worker thread. */
     public void poll(long timeoutMs) {
         try {
+            rewind();
+            long backOffMs = context.backoff();
+            timeoutMs = Math.max(timeoutMs, backOffMs);
```

Comment:
Shouldn't this be `Math.min`? The timeout passed into this poll method includes at least the next commit time. If you use `Math.max`, won't we miss that timeout if the `backoffMs` value is too large? (or vice versa)